### PR TITLE
5836 - Accordion protractor to puppeteer tests

### LINKED ISSUE #5836: Accordion: Convert Protractor Tests to Puppeteer
**Is your feature request related to a problem or use case? Please describe.**
Convert accordion component tests to puppeteer.

**Describe the solution you'd like**
Convert all tests scenarios in protractor to puppeteer. For the naming can be, `{component}.puppeteer-spec.js`

**Describe alternatives you've considered**
N/A

**Additional context**
For the visual regression tests, no need to convert and leave it as is for the meantime.

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Converted protractor test cases to puppeteer.

**Related github/jira issue (required)**:

Closes #5836 

**Steps necessary to review your pull request (required)**:

- Pull this branch
- Run puppeteer tests

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.

### CODE REVIEW

File: test/components/accordion/accordion.puppeteer-spec.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  const baseUrl = 'http://localhost:4000/components/accordion';
+
+  describe('Allow One Pane', () => {
+    const url = `${baseUrl}/example-allow-one-pane`;
+
+    beforeEach(async () => {
+      await page.goto(url, { waitUntil: ['domcontentloaded', 'networkidle2'] });
+    });
+
+    it('Should only allow one pane open at a time', async () => {
```
Comment (ericangeles) [inline test/components/accordion/accordion.puppeteer-spec.js:11]: For consistency, we can start the word `should` for all `it() function`. e.g., `it('should only allow one pane open at a time', async () => {})`

File: test/components/accordion/accordion.puppeteer-spec.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
+describe('Accordion Puppeteer Test', () => {
+  const baseUrl = 'http://localhost:4000/components/accordion';
```
Comment (ericangeles) [inline test/components/accordion/accordion.puppeteer-spec.js:2]: 🍌

File: test/components/accordion/accordion.puppeteer-spec.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  });
+
+  describe('Panels', () => {
+    const url = `${baseUrl}/example-accordion-panels`;
+
+    beforeEach(async () => {
+      await page.goto(url, { waitUntil: ['domcontentloaded', 'networkidle2'] });
+    });
+
+    it('Should have panels', async () => {
```
Comment (ericangeles) [inline test/components/accordion/accordion.puppeteer-spec.js:29]: Can lowercase the first word `should`.

**Review by ericangeles — CHANGES_REQUESTED** [review]
I like the simplicity and readability of the script. Please see some of the comments request changes. 🍌

**Comment by ericangeles** [comment]
Can you also add changelog?

**Review by ericangeles — APPROVED** [review]

**Review by EdwardCoyle — APPROVED** [review]

**Pull request merged**